### ISSUE TITLE
Spell Checking for :multi map dispatch keys

### ISSUE THREAD

**Issue by ikitommi** [metosin/malli#253]
Like this:

```clj
(-> [:multi {:dispatch :Type, :lazy-refs true}
     "AWS::ApiGateway::Stage"
     "AWS::ApiGateway::UsagePlan"
     "AWS::AppSync::ApiKey"]
    (m/explain
      {:Type "AWS::ApiGatway::UsagePlan"
       :Description "kikka"
       :UsagePlanName "kukka"})
    (me/with-spell-checking)
    (me/humanize))
; => {:Type ["did you mean AWS::ApiGateway::UsagePlan"]}
```

**Comment by ikitommi** [comment]
Impl mostly done, will add tests and ship it.

**Comment by ikitommi** [comment]
fixed in #252
